applications/NXsas.nxdl.xml

```diff
@@ -29,162 +29,166 @@
+          </doc>
         </field>
-        <field name="rotation_angle" type="NX_FLOAT" units="NX_ANGLE"></field>
-        <field name="aequatorial_angle" type="NX_FLOAT"
-          units="NX_ANGLE">
+        <field name="beam_center_y" type="NX_FLOAT" units="NX_LENGTH" minOccurs="0">
+          <doc>
+            This is the y position where the direct beam would hit the detector.
+            This is a length, not a pixel position, and can be outside of the
+            actual detector.
+
+            It is expected that data reduction will determine beam center from
+            the raw data, thus it is not required here.  The instrument can
+            provide an initial or nominal value to advise data reduction.
+          </doc>
         </field>
-		<field name="beam_center_x" type="NX_FLOAT" units="NX_LENGTH">
-			<doc>
-				This is the x position where the direct beam would hit the detector. This is a
-				length, not a pixel position, and can be outside of the actual detector.
-			</doc>
-		</field>
-        <field name="beam_center_y" type="NX_FLOAT" units="NX_LENGTH">
-			<doc>
-				This is the y position where the direct beam would hit the detector. This is a
-				length, not a pixel position, and can be outside of the actual detector.
-			</doc>
-		</field>
       </group>
       <field name="name" type="NX_CHAR">
-        <doc>Name of the instrument actually used to perform the experiment</doc>
+        <doc>Name of the instrument actually used to perform the experiment.</doc>
       </field>
     </group>
-    <group type="NXsample" name="sample">
+    <group type="NXsample" minOccurs="0">
       <field name="name">
-        <doc>Descriptive name of sample</doc>
-      </field>
-      <field name="aequatorial_angle" type="NX_FLOAT"
-        units="NX_ANGLE">
+        <doc>Descriptive name of sample.</doc>
       </field>
+      <field name="aequatorial_angle" type="NX_FLOAT" units="NX_ANGLE" minOccurs="0" />
     </group>
-    <group type="NXmonitor" name="control">
+    <group type="NXmonitor" minOccurs="0">
       <field name="mode">
         <doc>
-			Count to a preset value based on either clock time 
-			(timer) or received monitor counts (monitor). 
+          Count to a preset value based on either clock time
+          (timer) or received monitor counts (monitor).
         </doc>
         <enumeration>
           <item value="monitor"/>
           <item value="timer"/>
         </enumeration>
       </field>
       <field name="preset" type="NX_FLOAT">
-        <doc>preset value for time or monitor</doc>
+        <doc>Preset value for time or monitor.</doc>
       </field>
       <field name="integral" type="NX_FLOAT" units="NX_ANY">
-        <doc>Total integral monitor counts</doc>
+        <doc>Total integral monitor counts.</doc>
       </field>
     </group>
-    <group type="NXdata" name="data">
+    <group type="NXdata">
       <link name="data" target="/NXentry/NXinstrument/NXdetector/data"></link>
```

Comment:
With removal of `signal="1"` (above), we should indicate the plottable data here using the `signal` attribute of the NXdata group.  Yet, this is not required.